cishouseholds/pipeline/pipeline_stages.py

```diff
@@ -906,21 +914,21 @@ def report(
 
 
 @register_pipeline_stage("lab_report")
-def lab_report(survey_responses_table: str, swab_report_table: str, blood_report_table: str) -> DataFrame:
+def lab_report(input_survey_table: str, swab_report_table: str, blood_report_table: str) -> DataFrame:
     """Generate reports of most recent 7 days of swab and blood data"""
-    survey_responses_df = extract_from_table(survey_responses_table).orderBy("file_date")
+    survey_responses_df = extract_from_table(input_survey_table).orderBy("file_date")
     swab_df, blood_df = generate_lab_report(survey_responses_df)
     update_table(swab_df, swab_report_table, "overwrite")
     update_table(blood_df, blood_report_table, "overwrite")
 
 
 @register_pipeline_stage("record_level_interface")
```

Comment:
I think that this stage was replaced in terms of functionality by the raw_value_editing_config on the input stages - haven't found a single config that uses this stage.